### DISCUSSION TITLE
Using File Dialogs

### DISCUSSION THREAD

**Discussion by Dimitris-Toulis** [tauri-apps/tauri#3567] [Q&A] [answered]
I use a dialog to let the user select a file then i want to copy that file in order to store it. However I can't copy arbitrary files due to the scope settings. How can file dialogs be used if you can't use the selected file?

**Comment by FabianLars** [comment, marked as answer]
We are working on some kind of whitelist system for user selected paths. Until then you need to do file manipulation like this in rust where scope and traverse restrictions don't apply.
